scripts/create_joined_view.py

```diff
@@ -396,7 +397,40 @@ def main(args):
                                     assignment_fields + assignment_type_fields+ worker_fields,
                                     dispatcher_fields)
     logger.info(f"Final Item: {final_item.title}")
-    logger.info("Completed")
+    if args.create_dashboard:
+        logger.info("Creating dashboard")
+        
+        # create new webmap
+        map_item = project.dispatcher_webmap.save(
+            item_properties={"title": project.title + " Dashboard Map", "tags": [], "snippet": "Dashboard Map"})
+        new_webmap = WebMap(map_item)
+        
+        # swizzle in joined layer instead of assignments layer
+        for i, layer in enumerate(new_webmap.layers):
+            if layer["id"] == "Assignments_0":
+                new_webmap.remove_layer(layer)
+                new_webmap.add_layer(final_item)
```

Comment:
This will add it to the top of the map, is that what you want? Generally assignments should be below Workers.

Reply:
Yeah since this is a copy of the WF map I'm ok with it - clone_items and dashboard appears to key off layer id rather than layer order